tests/main/confinement-classic/test-snapd-hello-classic/Makefile

```diff
@@ -7,6 +7,11 @@ arch := $(shell $(CC) -dumpmachine)
 ifeq ($(arch),x86_64-linux-gnu)
 snap_arch = amd64
 dynamic_linker=ld-linux-x86-64.so.2
+else ifeq ($(arch),x86_64-suse-linux)
```

Comment:
Interesting, this code was designed to be built on Ubuntu as it then executes against the Ubuntu dynamic linker and libc. I wonder how stable this will be.